File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/ESurfaceContainerLowestDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ESurfaceContainerLowestDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd13
            AppTheme.STANDARD -> R.color.cstd13
            AppTheme.BLUE -> R.color.cbd13
            AppTheme.GREEN -> R.color.cgd13
            AppTheme.RED -> R.color.crd13
            AppTheme.NOTHING -> R.color.cnod13
        }
    }
}

File: lib/layer2_core/dark/e_surface_container_lowest_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class ESurfaceContainerLowestDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd13;
      case 'standard': return cstd13;
      case 'blue': return cbd13;
      case 'green': return cgd13;
      case 'red': return crd13;
      case 'nothing': return cnod13;
      default: return cstd13;
    }
  }
}
